xbmc/events/EventLog.cpp

```diff
@@ -199,7 +191,8 @@ void CEventLog::Remove(const std::string& eventPtrIdentifier)
 
   EventPtr eventPtr = itEvent->second;
   m_eventsMap.erase(itEvent);
-  std::remove_if(m_events.begin(), m_events.end(), [eventPtr](const EventPtr& otherEvent) { return eventPtr == otherEvent; });
+  auto it = std::remove_if(m_events.begin(), m_events.end(), [eventPtr](const EventPtr& otherEvent) { return eventPtr == otherEvent; });
```

Comment:
Yup. IIRC the initial code used to compare the identifier but then I implemented the comparison operator for events. Will update ASAP.